db/memtable.cc

```diff
@@ -600,6 +600,7 @@ struct Saver {
   bool* merge_in_progress;
   std::string* value;
   SequenceNumber seq;
+  std::string timestamp;
```

Comment:
Can `timestamp` be of type `std::string*`, like `value`?

Reply:
Yes, changed to pass in std::string*